apps/pie-storybook/stories/pie-modal.stories.ts

```diff
@@ -239,7 +237,6 @@ export const FocusManagement = createStory<ModalProps>(FocusableElementsPageStor
     returnFocusAfterCloseSelector: '#focus-3',
 });
 export const LargeTextContent = createBaseModalStory({
-    isFooterPinned: false,
```

Comment:
There was a bit of funkiness here. We were setting it to `false` but it was still `true` in the component, even though the storybook control showed `false`. You could toggle it off and on and it would work as expected but removing this means the control accurately reflects the current state.